### PR TITLE
clock

### PR DESCRIPTION
**Pull request by Artem18084**
- [DEMO LINK](https://Artem18084.github.io/layout_stop-watch/)

### CODE REVIEW

**Comment by Artem18084** [comment]
- [DEMO LINK](https://Artem18084.github.io/layout_stop-watch/)

**Comment by Artem18084** [comment]
- [DEMO LINK](https://Artem18084.github.io/layout_stop-watch/)

File: src/styles/main.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  height: 100vh;
+  position: relative;
+  transform: rotate(180deg);
+
+  &__arrow {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 5px;
+  }
```
Comment (AlinaKostiuk21) [inline src/styles/main.scss:11]:
```suggestion
  }
width: 5px;
  }
  
  &__arrow--min {
    height: 100px;
```
indentation missed

File: src/styles/main.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    animation: move 3600s steps(60) infinite;
+  }
+
+  &__arrow--sec {
+    background-color: red;
+    height: 300px;
+    transform-origin: center 0;
+    animation: move 60s steps(60) infinite;
+  }
+
```
Comment (AlinaKostiuk21) [inline src/styles/main.scss:25]: delete the indentation

**Review by VinnieJ-2k20 — APPROVED** [review]
I'll approve. Focus on other tasks.

**Pull request closed without merging**